evergreen/index.html

```diff
@@ -49,6 +51,47 @@ <h2>Contact Us</h2>
             <br/>
         </form>
 
+        <div id="passwordMessage"></div>
+
+
+        <script>
+            async function validate_pass() {
+
+
+                const password = document.getElementById("password").value;
+
+                const response = await fetch("{% url 'validate_pass' %}", {
+                    method: "POST",
+                    headers: {
+                        "Content-Type": "application/x-www-form-urlencoded",
+                    },
+
+                    body: `password=${encodeURIComponent(password)}`,
+                });
+
+                const result = await response.json();
+
+                const messageDiv = document.getElementById("passwordMessage");
+
+
+                // make it green is valid
+                if (result.valid_pass === "True") {
+
+                    messageDiv.style.color = "green";
+
+                    messageDiv.textContent = "VALID PASSWORD";
+
+
+                }
+                // make it red is BAD!!
+                else
+                {
+                    messageDiv.style.color = "red"
+                    messageDiv.textContent = "BAD PASSWORD"
```

Comment:
This should all be in the Javascript file and we should explicitly say what the rule is (tell user they need >=12 char)

Reply:
Done